package.json

```diff
@@ -191,15 +191,11 @@
     "ts-node": "~7.0.0",
     "tslint": "~5.11.0",
     "typescript": "~3.2.2",
+    "uuid": "^8.3.1",
     "webpack": "^4.41.5",
     "webpack-cli": "^3.3.10",
     "webpack-node-externals": "^1.7.2",
-    "webpack-source-map-support": "^2.0.1",
-    "base-64": "^0.1.0",
-    "node-fetch": "^2.6.1",
-    "mocha": "^8.1.1",
```

Comment:
Can you make sure that node-fetch and mocha are still in the package.json please?

Reply:
node-fetch and mocha appear as devDependencies. isomorphic-fetch has a dependency on node-fetch so I assume that is why node-fetch has been removed from the dependencies? As for mocha - it's a testing tool so is that why it has automatically been removed from dependencies? Following on from our conversation we agreed we should check it does not break anything prior to QA.